website/docs/reference/project-configs/config-version.md

```diff
@@ -2,6 +2,12 @@
 datatype: integer
 ---
 
+<VersionBlock firstVersion="1.5">
+
+Starting in dbt v1.5, the `config-version:` tag is optional.
```

Comment:
I'm not sure what this means for Cloud, we should find the right people to ask, though. This doesn't really change anything other than to make this version feature optional (previously if it wasn't included it would assume syntax was <v1 but those have all been deprecated)